src/stylesheets/settings/_settings-typography.scss

```diff
@@ -72,6 +72,19 @@ Accepts global, scoped, or none
 
 $theme-content-styles: 'scoped' !default;
 
+/*
+----------------------------------------
+Use Rails font pipeline
+----------------------------------------
+If true, outputs @font-face rules with
+`font-url`.
+----------------------------------------
+** Currently not working **
+----------------------------------------
+*/
+
+// $theme-use-rails-pipeline: false !default;
```

Comment:
Did you want to try to resolve this or leave it out?

Reply:
I don't know why I can't make this work — I guess we should leave the var out, but leave in the TODO in the mixin, and make a real issue around this